### DISCUSSION TITLE
[firebase_ui_auth] Add space between email and oauth widgets.

### DISCUSSION THREAD

**Discussion by bakua** [firebase/flutterfire#10605] [Feature request]
Hello,

can we have the option to set a space between the email Sign in button and the OAuth buttons, please? The current spacing just doesn't seem right.

<img width="472" alt="image" src="https://user-images.githubusercontent.com/494667/225257523-80094ff9-a8a8-40ea-a52b-3fa06d8bcccf.png">

**Comment by gazb23** [comment]
Unfortunately it looks even worse if you use `OAuthButtonVariant.icon` as there is no padding.

**Comment by timmolter** [comment]
Did anyone ever figure out how to add more space?
